docs/guides/primitive-input-output.ipynb

```diff
@@ -0,0 +1,472 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "e8a8a4b1-2ed2-4b82-8d02-98174a6da903",
+   "metadata": {},
+   "source": [
+    "# Primitive inputs and outputs\n",
+    "\n",
+    "This page gives an overview of the inputs and outputs of the Qiskit Runtime V2 primitives that execute workloads on IBM Quantum&trade; compute resources. These primitives provide you with the ability to efficiently define vectorized workloads by using a data structure known as a **Primitive Unified Bloc (PUB)**. These PUBs are the fundamental unit of work a QPU needs to execute these workloads. They are used as inputs to the [`run()`](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.EstimatorV2#run) method for the Sampler and Estimator primitives which execute the defined workload as a job. Then, after the job has completed, the results are returned in a format that is dependent on both the PUBs used as well as the runtime options specified from the Sampler or Estimator primitives."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "04b9204f-f058-433b-89f9-05df14645e27",
+   "metadata": {},
+   "source": [
+    "## Overview of PUBS\n",
+    "\n",
+    "When invoking a primitive's [`run()`](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.EstimatorV2#run) method, the main argument that is required is a `list` of one or more tuples -- one for each circuit being executed by the primitive. Each of these tuples is considered a PUB and the required elements of each tuple in the list will depend on the the primitive used. These data provided to these tuples can also be arranged in a variety of shapes to provide flexibility in a workload through broadcasting -- the rules of which are described in the [section below](#broadcasting-rules).\n",
+    "\n",
+    "### Estimator PUB\n",
+    "For the Estimator primitive, the format of the PUB should contain at most four values:\n",
+    "- A single `QuantumCircuit`, which may contain one or more [`Parameter`s](../api/qiskit/qiskit.circuit.Parameter)\n",
+    "- A list of one or more observables, which specify the expectation values to estimate, arranged into an array (e.g. a single observable represented as a 0-d array, a list of observables as a 1-d array, etc). The data can be in any one of the `ObservablesArrayLike` format such as `Pauli`, `SparsePauliOp`, `PauliList`, or `str`.\n",
+    "- A collection of parameter values to bind the circuit against. This can be specified as a single array-like object where the last index is over circuit `Parameter`s, or omitted (or equivalently, set to `None`) if the circuit has no `Parameter`s.\n",
```

Comment:
```suggestion
    "- A list of one or more observables, which specify the expectation values to estimate, arranged into an array (for example, a single observable represented as a 0-d array, a list of observables as a 1-d array, and so forth). The data can be in any one of the `ObservablesArrayLike` format such as `Pauli`, `SparsePauliOp`, `PauliList`, or `str`\n",
    "- A collection of parameter values to bind the circuit against. This can be specified as a single array-like object where the last index is over circuit `Parameter`s, or omitted (or equivalently, set to `None`) if the circuit has no `Parameter`s\n",
```

Reply:
Resolved this with a slightly different edit to:

"(for example, a single observable represented as a 0-d array, a list of observables as a 1-d array, and so on)"